private/buf/bufgen/config.go

```diff
@@ -339,3 +370,8 @@ type readConfigOptions struct {
 func newReadConfigOptions() *readConfigOptions {
 	return &readConfigOptions{}
 }
+
+func pluginNameIsRemote(pluginName string) bool {
+	// Local plugins cannot contain "/"
+	return strings.Contains(pluginName, "/")
+}
```

Comment:
This looks unused (I imagine it was kept around before we started using `bufplugin.ParsePluginPath`).